go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/go-kit/kit v0.9.0
 	github.com/google/uuid v1.1.1
 	github.com/lestrrat-go/jwx v0.9.0
-	github.com/optimizely/go-sdk v1.6.2-0.20210507191537-aba050a09690
+	github.com/optimizely/go-sdk v1.6.2-0.20210903184112-9d97c16f5787
```

Comment:
this should be update with the go release.

Reply:
reverting it will cause the unit tests to fail.